oscLib.js

```diff
@@ -41,6 +41,14 @@ function Client() {
   } 
 }
 
+function makeNote(note=60,dur=0.5) {
+  let env = new p5.Envelope(0.01,dur,0.0,0.0);
+  let triOsc = new p5.Oscillator('triangle');
```

Comment:
Oscillator is created again everytime this function is called, poor garbage-collector is making over hours

Reply:
Yeah, I know.. but this is the easiest solution to avoid a lot of 'gedoe' for the students.

Reply:
Alright forwards then!